server/src/resolvers/Query.ts

```diff
@@ -0,0 +1,76 @@
+import { PrismaClient } from "@prisma/client";
+
+export const Query = {
```

Comment:
In this file, you also have some examples for the above mentioned things (reuse, type defs)